docs/source/getting_started/teams.rst

```diff
@@ -0,0 +1,132 @@
+========
+Teams
+========
+
+Teams were introduced to simplify securely sharing your assets across projects. All assets
+are owned by Teams, including projects.
+
+.. warning::
+    Teams are not available on all Citrine Platform deployments.
+
+    Please contact support for more information.
+
+Basic Team Use
+-----------------
+
+In the following examples, ``citrine`` is the name of your :class:`~citrine.citrine.Citrine`
+client object.
+
+After connecting to the Citrine Platform, you'll most likely want to find a specific Team.
+To list all Teams of which you are a member, use the ``list`` method.
+
+.. code-block:: python
+
+    citrine.teams.list()
+
+There are a few ways to find a single Team from this list.
+
+.. code-block:: python
+
+    # 1. Iterate over the list of teams and filter by name.
+    team_name = "Team A"
+    all_teams = citrine.teams.list()
+    team_a = next((team for team in all_teams if team.name == team_name), None)
```

Comment:
This is a pattern we should discourage in general.  Our customers probably won't have many teams, but this pattern has been used with other resource types in the past, causing high load on our services due to executing a new SQL query for each page of results.  Given Casbin's abysmal performance, it would be nice to avoid this pattern for teams also.

Doing a search is much more efficient.  Searching for a team is implemented in the API but it looks like it isn't implemented in Citrine Python.  Something to add to the todo!  Given that we can't search yet, I think we have to leave this line as-is.

Reply:
I agree, so I just filed a ticket to that effect: https://citrine.atlassian.net/browse/PLA-9058

In the mean time, I think I'll remove this option in order to reduce the chance people will use it.